includes/data-stores/class-wc-payment-token-data-store-table.php

```diff
@@ -0,0 +1,256 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Payment Token Data Store: Custom Table.
+ *
+ * @version  2.7.0
+ * @category Class
+ * @author   WooThemes
+ */
+class WC_Payment_Token_Data_Store_Table implements WC_Payment_Token_Data_Store, WC_Object_Data_Store {
```

Comment:
Let's standardise our interface names. Abstracts we prefix, but interfaces I believe we should suffix. This also avoids conflicts if the actual store has the same name as the interface (e.g. for my customers one I named it WC_Customer_Data_Store since its not a CPT or custom table).

So I'd go with WC_Payment_Token_Data_Store_Interface, and file name class-wc-payment-token-data-store-interface.